components/homme/src/share/cxx/EulerStepFunctorImpl.hpp

```diff
@@ -994,29 +1000,66 @@ VECTOR_SIMD_LOOP
         }
       }
     }
+
+    forij {
+      VECTOR_SIMD_LOOP forlev {
+        x(i,j,lev) *= dpmass(i,j,lev);
+      }
+    }
   } else if (limiter_option == 9) {
+    ViewUnmanaged<Real[NP][NP][NUM_PHYSICAL_LEV]>
+      ptens(&iptens(0,0,0)[0]);
+    ViewUnmanaged<Real[NP][NP][NUM_PHYSICAL_LEV]>
```

Comment:
This doesn't have to be, but do so for alignment.

Reply:
Isn't it already aligned?